test/integration/connection-test.js

```diff
@@ -289,6 +289,7 @@ describe('Initiate Connect Test', function() {
 
     // Specify a cipher that should never be supported by SQL Server
     config.options.cryptoCredentialsDetails = {
+      maxVersion: 'TLSv1.2',
```

Comment:
I think we should move this out of the tests somewhere else. This is going to fix the test failures, but I think we should make sure that we only ever use `TLSv1.2` and never `TLSv1.3` inside tedious when TDS < 8.0 is used - and only support `TLSv1.3` once we add TDS 8.0 support later.

Reply:
Just committed some changes. Was this closer to what you had in mind?